sdk/contentsafety/azure-ai-contentsafety/tests/test_content_safety.py

```diff
@@ -1,5 +1,10 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
 import os
-import base64
 import functools
 
 from devtools_testutils import AzureRecordedTestCase, PowerShellPreparer, recorded_by_proxy
```

Comment:
It will still work as an alias, but yes -- you should import the `EnvironmentVariableLoader` instead: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/tests.md#deliver-environment-variables-to-tests

Reply:
Fixed.